.azure-pipelines/linux-conda-CI.yml

```diff
@@ -113,7 +129,13 @@ jobs:
           python setup.py install
           cd ..
         else
-          pip install onnx$(onnx.version)
+          if [ '$(onnx.version)' == 'test' ]
```

Comment:
A trick of the test pypi issue is to install a official release, then upgrade to the test package.

Reply:
I won't probably be able to cherry pick. I'll modify the CI file when onnx 1.7 is released.